server/monitor/src/main/java/org/apache/accumulo/monitor/rest/api/DeadLoggerList.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeadLoggerList {
```

Comment:
I wrote this one for backwards compatibility, since the current monitor makes a distinction between DeadLoggers and DeadServers (even though they have the same information), and I didn't want to have two `deadTabletServer` tags on the XML. Although I might be able to make another variables inside DeadServerList to use as DeadLoggers and remove this and the above class.